packages/checkout/widgets-lib/src/components/TokenImage/TokenImage.tsx

```diff
@@ -32,12 +32,17 @@ export function TokenImage({
     [src, error],
   );
 
+  const { ...cleanedProps } = forwardedProps as Record<string, unknown>;
+  if (Object.prototype.hasOwnProperty.call(cleanedProps, 'responsiveSizes')) {
+    delete cleanedProps.responsiveSizes;
+  }
+
```

Comment:
the `TokenImage` is a react component and is not using Biome, right? Why do we have to explicitly filter out `responsiveSizes`?

Reply:
ah, is it because `<MenuItem.FramedImage>` component `uses` it?

Reply:
**Good question**
This component is always used as the token image fallback for the MenuItem from Biome, and its this one sending the extra unexpected props.

```tsx
      <MenuItem.FramedImage
        circularFrame
        use={(
          <TokenImage src={icon} name={name} defaultImage={defaultTokenImage} />
        )}
      />
      ```

Reply:
makes sense, thanks J!